MarkBind/markbind
# Migrate moment to dayjs

**What is the purpose of this pull request?**




**Overview of changes:**
- Migrate moment to dayjs
- Add unit test for the date usage
- Update relevant docs

https://momentjs.com/docs/#/-project-status/recommendations/

**Anything you'd like to highlight/discuss:**


**Testing instructions:**
- Docs update: https://deploy-preview-2007--markbind-master.netlify.app/userguide/formattingcontents#dates
- Test cases added and should be checked by CI



**Proposed commit message: (wrap lines at 72 characters)**
Migrate moment to dayjs



---

**Checklist:** :ballot_box_with_check:
